jsapp/js/components/reports/reports.es6

```diff
@@ -0,0 +1,557 @@
+import React from 'react';
+import reactMixin from 'react-mixin';
+import autoBind from 'react-autobind';
+import Reflux from 'reflux';
+import {dataInterface} from 'js/dataInterface';
+import {actions} from 'js/actions';
+import {bem} from 'js/bem';
+import {stores} from 'js/stores';
+import ui from 'js/ui';
+import mixins from 'js/mixins';
+import DocumentTitle from 'react-document-title';
+import { txtid } from '../../../xlform/src/model.utils';
+import alertify from 'alertifyjs';
+import {launchPrinting} from 'utils';
+import {REPORT_STYLES} from './reportsConstants';
+import CustomReportForm from './customReportForm';
+import QuestionGraphSettings from './questionGraphSettings';
+import ReportContents from './reportContents';
+import ReportStyleSettings from './reportStyleSettings';
+
+export default class Reports extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      graphWidth: '700',
+      graphHeight: '250',
+      translations: false,
+      activeModalTab: 0,
+      error: false,
+      isFullscreen: false,
+      reportLimit: 200,
+      customReports: false,
+      showReportGraphSettings: false,
+      showCustomReportModal: false,
+      currentCustomReport: false,
+      currentQuestionGraph: false,
+      groupBy: '',
+    };
+    autoBind(this);
+  }
+
+  componentDidMount() {
+    this.loadReportData();
+    this.listenTo(actions.reports.setStyle, this.reportStyleListener);
+    this.listenTo(actions.reports.setCustom, this.reportCustomListener);
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.currentCustomReport !== prevState.currentCustomReport) {
+      this.refreshReportData();
+    }
+    if (this.state.groupBy !== prevState.groupBy) {
+      this.refreshReportData();
+    }
+  }
+
+  loadReportData() {
+    const uid = this.props.params.assetid || this.props.params.uid;
+
+    stores.allAssets.whenLoaded(uid, (asset) => {
+      let rowsByKuid = {};
+      let rowsByIdentifier = {};
+      let groupBy = '',
+          reportStyles = asset.report_styles,
```

Comment:
it looked like the linting expanded variable declarations from `let x, y` into `let x; let y;`. maybe a missed spot here?
same thing on line 150